salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc

```diff
@@ -0,0 +1,1164 @@
+	</flow>
+</mule>
+----
+
+=== Steps for Running This Example
+
+. Verify that your connector is configured.
+. Save the project.
+. Test the flow by sending a `GET` to
+`localhost:8081/profile/{DATA_MODEL_NAME}/{Id}/calculated-insights/{CALCULATED_INSIGHT_NAME}`.
+
+
+== Data Action Webhook
+
+This Mule flow shows how to create a webhook that will log its payload when triggered by a Data Action. Refer to the https://help.salesforce.com/s/articleView?language=en_US&type=5&id=sf.c360_a_data_actions.htm[Salesforce Data Action docs].
+
+This example uses the following operations:
+
+* *Data Action Webhook* +
+Creates a webhook as the target of a Salesforce CDP Data Action.
+* *Logger* +
+Shows the payload sent to the Data Action Webhook.
+
+image::data-action-webhook.png[Salesforce CDP Data Action Webhook Flow Diagram - (CDP Data Action Webhook - Logger Log Payload)]
+
+=== XML for This Example
+
+Paste this code into the Studio XML editor to quickly load the flow for this example into your Mule app:
+
+[source,xml,linenums]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:sdc="http://www.mulesoft.org/schema/mule/sdc"
+	xmlns:http="http://www.mulesoft.org/schema/mule/http"
+    xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="
+    http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+    http://www.mulesoft.org/schema/mule/sdc http://www.mulesoft.org/schema/mule/sdc/current/mule-sdc.xsd">
+	<flow name="Data_Action_Webhook" doc:id="3072197f-ee1f-4ed5-99d2-a9fa62230dbf" >
+		<sdc:webhook-listener doc:name="Data Action Webhook" doc:id="fe3a924a-ba94-447d-9d3c-df079fc01de2" config-ref="Salesforce_CDP_Sdc_webhook_config" path="${webhook.path}" signingKey="${webhook.signingKey}" signingAlgorithm="HmacSHA256"/>
+		<logger level="INFO" doc:name="Log Payload" doc:id="716538d3-7d06-4e9c-a4f7-b0d350b4dafe" message="#[output application/json --- payload]"/>
+	</flow>
+</mule>
+----
+
+=== Steps for Running This Example
+
+. Verify you have https://help.salesforce.com/s/articleView?id=sf.c360_a_create_streaming_insight.htm&type=5[created a Streaming Insight].
+. Verify you https://help.salesforce.com/s/articleView?id=sf.c360_a_data_actions.htm&type=5[created a Data Action Target] with an Action Target Type of `Webhook`.
+. Verify you https://help.salesforce.com/s/articleView?id=sf.c360_a_data_actions.htm&type=5[created a Data Action] running on a condition against your streaming Calculated Insight, that will use the Data Action Target.
```

Comment:
This text is confusing. Maybe:
Verify that you https://help.salesforce.com/s/articleView?id=sf.c360_a_data_actions.htm&type=5[created a Data Action that uses the Data Action Target and that runs on a condition against your streaming Calculated Insight.

(I am not sure if my rewrite makes sense in the context of the product.) What does it mean to "run on a condition"?